_i18n/en.yml

```diff
@@ -326,13 +338,28 @@ help_pages:
 
       [Why do I need to use login.gov to access government services online?](site.baseurl/help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/)
     what-is-my-personal-key: |-
-      A personal key is a 16-character secret code we give you when you create an account. The personal key is the only way to sign in if you lose access to the device you use for two-factor authentication.
+      A personal key is a 16-character secret code we have been historically generating as a form of two factor authentication. [Read more about two-factor authentication here](site.baseurl/help/creating-an-account/what-is-two-factor-authentication/)
```

Comment:
two-factor instead of two factor. there are a few in the doc but I won't tag them all

Reply:
Fixed all instances of "two factor"